rick_and_morty_character.py

```diff
@@ -0,0 +1,145 @@
+import json
+
+
+url = 'https://rickandmortyapi.com/api/character/'
+
+def get_characters():
+    import requests
+
+    return requests.get(url).json()['results']
+
+try:
+    with open ("saved_characters.json", "r") as file:
+        saved_characters_r = json.load(file)
+except FileNotFoundError:
+    with open("saved_characters.json", "w") as file:
+        json.dump(get_characters(), file, indent=4, ensure_ascii=False)
+
+def info_about_character():
+    info_about_characters = saved_characters_r
+    names = []
```

Comment:
```suggestion
    characters = []
    
    for i in info_about_characters:
        characters.append({
            'name': i['name'],
            'status': i['status'],
            'species': i['species'],
            'gender': i['gender'],
            'location': i['location']['name']
        })
        
        return characters
```

это код, который работает аналогично коду в строках 20-45

Reply:
понял, спасибо

Reply:
def info_about_character():
    list_info_about_characters = []
    for characters in saved_characters_r:
        list_info_about_characters.append({
            'name': characters['name'],
            'status': characters['status'],
            'species': characters['species'],
            'gender': characters['gender'],
            'location': characters['location']['name']
        })
    return list_info_about_characters